tests/ui/pin_project/add-attr-to-struct.stderr

```diff
@@ -1,4 +1,4 @@
-error: duplicate #[pin] attribute
+error: expected attribute arguments in parentheses: `pin(...)`
```

Comment:
Should I update comments like this to something more appropriate?
https://github.com/taiki-e/pin-project/blob/ce860dd624aeabede645f38745aa9af334e559f0/tests/ui/pin_project/add-attr-to-struct.rs#L7

Reply:
Yes, please update these comments to match new error messages.

Reply:
Done.
I tried to keep the style consistent, let me know if you want me to change something.